### DISCUSSION TITLE
Unable to flash using web - sonoff basic R1

### DISCUSSION THREAD

**Discussion by drumstick77** [arendst/Tasmota#19127] [Q&A]
Hi All,

I tried using the web tool to flash, but it's not working at all. 
it's broadcasting AP but I cannot latch onto it, not getting any IP address. Any advise on this? thanks

**Comment by fragtion** [comment]
Same problem here. Web flash times out. Need to use the upload function (which also only seems to work after the second or third attempt)

I'm also having wifi connectivity issues after resetting the device, which fresh flash+erase doesn't resolve.
Basically, it goes into AP mode fine, but won't connect to any other AP. Says to check credentials.. I tried entering the ssid and password manually via the console but still no luck
I thought it was Issue #391 at first, and ended up destroying one of my switches while trying to mess around with resistors etc. Then the same thing happened to another switch (yes I have plenty of them...)
This after upgrading to 13.0.0

**Reply by nickshew** [reply]
FYI - I had a problem after flashing a Sonoff Basic R1 with Tasmota 13.2.0.  Getting 10 192.168.4.1 was difficult (couldn't on my windows PC but did on my iPhone).  The device was unstable and unreliable.  I was about to give up when I decided to flash v9.5 instead.  It worked fine and in now installed doing its job.  I had the 13.2.0 problem on 2 different R1s.  One was a stock standard and the other (now in use) was one I had modified with 4Mb flash.  This would seem to be a v13 issue and not a flash size issue.

**Reply by sfromis** [reply]
No, it is not a Tasmota version 13 issue, it is due to the crappy hardware barely able to power itself. And Tasmota does draw a bit more power.

**Reply by nickshew** [reply]
The Tasmota versions are quite different on power usage then. I tried lite with the same result.  Any idea if the later Sonoff Basic revisions are better?
